README.md

```diff
@@ -97,3 +97,16 @@ require __DIR__.'/../config/bootstrap.php';
 $kernel = new \App\Kernel($_SERVER['APP_ENV'], (bool) $_SERVER['APP_DEBUG']);
 return new \Symfony\Bundle\FrameworkBundle\Console\Application($kernel);
 ```
+
+You may then encounter an error with PhpParser. If this is the case, you should create a new environment for your application that will disable inlining. In `config/packages/phpstan_env`:
```

Comment:
1) Can you post the full error message that user might encounter?
2) The file path `config/packages/phpstan_env` is probably missing a file extension (`yml`?).